### DISCUSSION TITLE
User flow for digital/electronic locker reference solution

### DISCUSSION THREAD

**Discussion by jayadinni** [Sunbird-RC/community#47] [Show & Tell]
@pramodkvarma @jagadishbabu
Added the end user flow for digital locker reference solution:
https://docs.sunbirdrc.dev/example-use-cases/digital-locker/digital-locker-user-flow
Please review and provide your inputs.
cc: @dileepbapat @parthlawate

**Comment by pramodkvarma** [comment]
Nice one @jayadinni. We should ensure the issuer flow is clearly separated out from the core "locker" flows.

**Comment by jayadinni** [comment]
The [issuer flow](https://docs.sunbirdrc.dev/example-use-cases/digital-locker/digital-locker-user-flow#user-flows-issuer) would be same for other use cases (Edu, vaccination,...) which has 3 types of issuance. There is nothing specific to 'locker' use case. @pramodkvarma is the understanding correct?

**Comment by pramodkvarma** [comment]
Yes, understanding is correct. But, make sure someone wanting to adopt digital locker solution should have this documented without getting confused about other sample solutions. 

Also, Digital/electronic locker itself doesn't have issuance, it is an independent module that can be offered to issuers by digital locker platform provider "as an option". Issuers can choose to have their own issuer platform which issues credentials and published to digital locker. This separation need to be be clear.
